packages/@azure/servicebus/data-plane/test/testUtils.ts

```diff
@@ -383,3 +384,18 @@ export async function purge(
     }
   }
 }
+
+// Maximum wait duration for the Streaming Receiver to receive the messages = `10000 ms`(10 seconds)(= maxWaitTime)
+// Keep checking whether the predicate is true after every `1000 ms`(1 second) (= delayBetweenRetries)
```

Comment:
Use the jsdoc structure of `/** ... */ to describe the function, otherwise we wont see this comment when hovering on the function when it is used